content/millennium/r4/financial/charge-item.md

```diff
@@ -106,6 +111,17 @@ Notes:
 <%= headers status: 200 %>
 <%= json(:r4_chargeitem_bundle) %>
 
+### Example - Search by Context
```

Comment:
Should we get rid of the Search by Id example all together since it is pretty much the exact same as a Retreive by Id?

Reply:
+1

Reply:
removed it here https://github.com/cerner/fhir.cerner.com/pull/665/commits/52ec1eb5766e51bfbe059e26fbe147fab674940c#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbL103-L114

Reply:
Thanks for making that change.